src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -32,50 +33,28 @@
 	private static final Logger LOGGER = LoggerFactory.getLogger(JsonConvert.class);
 
 	/**
-	 * The default path to JSON file.
+	 * The path to export a list of apartments in a JSON file.
+	 *
+	 * @return <i>Path</i> of the JSON file at the root of the project
 	 */
-	public static final Path APARTMENT_PATH_JSON = Path.of("Apartment_Json.json");
+	private static Path getUniqueExportPath() {
+		return Path.of("exportApartments" + System.currentTimeMillis() + ".json");
+	}
 
 	/**
 	 * The method return a default JSON file read by jsonToApartments.
 	 * 
 	 * @return <i>Path</i> where jsonToApartments will read.
-	 * @throws URISyntaxException if the resource cannot be found
 	 */
-	private static final Path startApartment() {
+	private static Path startApartments() {
 		try {
 			URI ressource = JsonConvert.class.getResource("defaultJsonToApartments.json").toURI();
```

Comment:
Je trouverais le nom de fichier `defaultApartments.json` plus clair. OPT

Reply:
Fix in commit 7339ee9399a3adde961911951489d01a498b746c